session.py

```diff
@@ -0,0 +1,671 @@
+import requests
+
+import json
+
+import toml
+
+from requests.sessions import session
+
+class mySession:
+
+    s = requests.session()
+
+    #baseURL
+    
+    base_URL = "https://wger.de/api/v2/"
+
+
+    #loginURL
+
+    loginURL="https://wger.de/en/user/login"
+
+
+    #baseReferers
+
+    base_workoutReferer = "https://wger.de/en/workout/"
+    base_nutritionReferer = "https://wger.de/en/nutrition/"
+
+
+    def __init__(self):
+        cookies = self.login_and_cookies()
+        
+        #extract csrftoken and sessionId
+        self.csrftoken = cookies[0]
+        self.sessionId = cookies[1]
+
+        #create login header
+        self.headers = {
+        'Content-Type': 'application/json',
+        'Referer': '',
+        'X-CSRFToken' : self.csrftoken,
+        'Authorization' : 'Token bd9ecc45b397f0e22bd0a8bd3304010ab4b586d8',
+        'Cookie':f'csrftoken={self.csrftoken}; sessionid={self.sessionId}'
+    }
+
+    @classmethod
+    def show_request_details(cls, response_list):
+        """ Print the status_code and the content of a request and
+            whether the request was made with success
+
+            response_list ([request, mySession.check(request)])
+
+        """
+        print(response_list[1])
+        print(f"status code: {response_list[0].status_code}")
+        print(f"response : {response_list[0].content}")
+        print("\n")
+    
+    def login_and_cookies(self):
+        """ Returns the csrftoken and sessionId of a session
+        """
+        self.s.get(self.loginURL)
+        csrftoken = self.s.cookies.get('csrftoken')
+        sessionId = self.s.cookies.get('sessionid')
+        self.payload = {
+            'username': 'tiberiu1213',
+            'password' : 'THeoTHEo2013&',
+            'submit' : 'Login',
+            'csrfmiddlewaretoken':csrftoken
+            }
+        self.loginResponse = self.s.post(self.loginURL,data = self.payload, headers=dict(Referer=self.loginURL))
+        return [csrftoken,sessionId]
+
+    #requests
+
+    def get(self, type, id = None):
+        """ Makes any get request
+            
+            args:
```

Comment:
You can use the following format for describing parameters of classes/methods/functions:
:param <parameter_name>: description